library/CMakeLists.txt

```diff
@@ -189,21 +199,27 @@ if(USE_SHARED_MBEDTLS_LIBRARY)
     add_library(mbedcrypto SHARED ${src_crypto})
     set_target_properties(mbedcrypto PROPERTIES VERSION 2.21.0 SOVERSION 4)
     target_link_libraries(mbedcrypto ${libs})
-    target_include_directories(mbedcrypto
-        PUBLIC ${MBEDTLS_DIR}/include/)
+    target_include_directories(mbedcrypto 
+        PUBLIC
+            $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}/../include>
```

Comment:
What's `$<…>`? I can't find it in https://cmake.org/cmake/help/v3.0/manual/cmake-language.7.html#syntax

Reply:
https://cmake.org/cmake/help/v3.3/manual/cmake-generator-expressions.7.html

Reply:
Thanks!